src/temp/stories/ProjectFilter.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import Filter from '../Project/Filter';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof Filter> = {
+  component: Filter,
+};
+
+export default meta;
+type Story = StoryObj<typeof Filter>;
+
+export const FilterInActive: Story = {
+  args: {
+    activeFilter: false,
+    projectList: [],
+  },
+};
```

Comment:
Typo here - this should be `FilterInactive` as `inactive` is one word. This will help storybook split the story name correctly as well, as it is dependent on case.